src/binding/actix/server_request.rs

```diff
@@ -156,23 +156,18 @@ mod tests {
     use super::*;
     use actix_web::test;
 
+    use crate::test::*;
```

Comment:
This is why I said we could do away with the "pub use". I was thinking clients would `use crate::test::fixtures::*` instead of `use crate::test_fixtures::*`.

Reply:
Hm, the mod is actually called `test` now. `fixtures` is submodule with actual examples only used internally by `v03` and `v10` submodules. 

So actual tests would actually use `crate::test::v03::minimal()` in some form.

For example:
```
use crate::test;

test::v03::minimal()
```



I also just noticed that `test` mod name can clash for example with actix test, like

```
use actix_web::test;
use crate::test
```

So what we can do is actually create `test::fixtures` module that would be used like you showed. That would allow us to add more testing modules/utils later on and help with the conflicts?

Reply:
Sorry, I was confused earlier. So, are you suggesting that we have `test::fixtures::{v03,v10}`? With that, we could move all the stuff that used to be in `test_data/data.rs` to `test/mod.rs`. Is that what you're thinking?

Reply:
Yeah, something along those lines. I was originally thinking to leave `data.rs` (and name it as we agree) but move that to the `mod.rs` as well.

Reply:
@jcrossley3 OK I think I captured this idea in the latest push. Let me know how it looks to you.